build.sbt

```diff
@@ -137,6 +137,22 @@ scalacOptions ++= {
 
 Test / parallelExecution := false
 Test / fork := true
+import Tests._
+{
+  def groupByFirst(tests: Seq[TestDefinition]) =
```

Comment:
Not sure I understand the problem. How did your change touch the Registry?

Reply:
Here is the official documentation of SBT: https://www.scala-sbt.org/1.x/docs/Testing.html#Forking+tests